src/components/MenuBar.vue

```diff
@@ -284,8 +287,14 @@ export default {
 	},
 	beforeDestroy() {
 		window.removeEventListener('resize', this.getWindowWidth)
+		unsubscribe('files:sidebar:opened', this.redrawAfterTransition)
+		unsubscribe('files:sidebar:closed', this.redrawAfterTransition)
 	},
 	methods: {
+		redrawAfterTransition() {
+			// wait for transition to complete (100ms)
+			setTimeout(this.redrawMenuBar, 110)
```

Comment:
It's somewhat funny that this is necessary. Files sidebar is based on nc vue and that says 'opened' and 'closed' are emitted AFTER the transition:
https://github.com/nextcloud/nextcloud-vue/blob/e6bb18756ebf2c00d4f7e37fe7faaa4340ef8c13/src/components/AppSidebar/AppSidebar.vue#L439
But if i don't put a delay redraw menu bar is called when the menubar still has the old size.